benchalerts/clients.py

```diff
@@ -178,6 +237,11 @@ def get_comparison_to_baseline(self, contender_sha: str) -> dict:
             The SHA of the contender commit to compare. Needs to match EXACTLY what
```

Comment:
Sorry I didn't catch this originally but I wonder if it is worth being really obvious about this. 
```suggestion
            The commit SHA of the contender commit to compare. Needs to match EXACTLY what
```